config/sys.config

```
@@ -100,6 +100,8 @@
                       'CN779' => [779.5, 779.7, 779.9],
                       'AU915' => [916.8, 917.0, 917.2, 917.4, 917.5, 917.6, 917.8, 918.0, 918.2],
                       'AS923' => [923.2, 923.4, 923.6, 923.8, 924.0, 924.2, 924.4, 924.5, 924.6, 924.8],
```

```suggestion
                      'AS923-1' => [923.2, 923.4, 923.6, 923.8, 924.0, 924.2, 924.4, 924.6],
```

Reduce to 8 channels and rename to AS923-1